### DISCUSSION TITLE
How to add scrolled ListView(or Column) with another widget in the Row?

### DISCUSSION THREAD

**Discussion by antonnurapkin** [flet-dev/flet#1232] [Q&A]
### Question

I wrote this code and now my list isn't scrollable

I want to have one scrolled list on the left and docs about any item on the right

### Code sample

```python
import flet as ft

def main(page: ft.Page):
page.title = "Auto-scrolling ListView"

col = ft.Column(scroll='ALWAYS',width=page.width*0.45)

for i in range(0, 60):
    col.controls.append(ft.TextButton(width=page.width*0.45,text=f"Line{i+1}"))

widgets = [
    # ft.TextButton(text='Расчёт на прочность болтвого соединения',width=page.width*0.45),
    col,
    ft.TextButton(text='Documentation',width=page.width*0.45)
]

row = ft.Row(spacing=0, controls=widgets,alignment = ft.MainAxisAlignment.SPACE_EVENLY)

page.add(row)`
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by gbrandt** [comment]
Column needs a fixed height to scroll.  Either by being forced into a height by the controls around it or by giving it a height.  The quick fix is to change `col = ft.Column(scroll='ALWAYS',width=page.width*0.45)` into `col = ft.Column(scroll='ScrollMode.ALWAYS, width=page.width*0.45, height = page.height)`.  You can then add a page.on_resize to change the width and height if the user changes the windows size.

**Reply by antonnurapkin** [reply]
Yoe're the Savior! Thanks!
